src/pal/src/CMakeLists.txt

```diff
@@ -238,10 +238,12 @@ add_library(coreclrpal
   ${PLATFORM_SOURCES}
 )
 
-add_library(tracepointprovider
-  STATIC
-  misc/tracepointprovider.cpp
-)
+if(CLR_CMAKE_PLATFORM_LINUX AND FEATURE_EVENT_TRACE)
```

Comment:
I think it would be good to have a comment here that describes that this line builds in the functionality required to load the LTTng tracepoints at runtime.

I probably should have added that myself when I added misc/tracepointprovider.cpp.

Reply:
Done